### PR TITLE
fix: waiting other frames.

### PR DESCRIPTION
**Pull request by mohammedzeglam-pg**

### CODE REVIEW

**Comment by Qizot** [comment]
Thanks for catching up the bug :) 

Since the sink can now send a single track only this `write_frame_interleaved` is no longer valid as we have to check if the tracks are present. I will create a different fix that will aim to do what you did.

File: lib/membrane_rtmp_plugin/rtmp/sink/sink.ex [hunk 1]
```diff
@@ -253,7 +253,11 @@ defmodule Membrane.RTMP.Sink do
     end
   end
 
-  defp write_frame_interleaved(%{frame_buffer: %{audio: audio, video: video}} = state)
+  defp write_frame_interleaved(
+         %{
+           frame_buffer: %{{Membrane.Pad, :audio, 0} => audio, {Membrane.Pad, :video, 0} => video}
```
Comment (Qizot) [inline lib/membrane_rtmp_plugin/rtmp/sink/sink.ex:258]:
```suggestion
           frame_buffer: %{Pad.ref(:audio, 0) => audio, Pad.ref(:video, 0) => video}
```

**Review by Qizot — CHANGES_REQUESTED** [review]
Thanks for catching up the bug :)

Can you bump the patch version as well in `mix.exs` and `README.md`?

File: lib/membrane_rtmp_plugin/rtmp/sink/sink.ex (hunk 1 above)
Comment (mohammedzeglam-pg) [reply lib/membrane_rtmp_plugin/rtmp/sink/sink.ex]: Thanks for help me with this. and sorry for take your time to guide me.

Comment (Qizot) [reply lib/membrane_rtmp_plugin/rtmp/sink/sink.ex]: No need to be sorry, you are more than fine :)

**Review by Qizot — APPROVED** [review]

**Pull request merged**